openssl-sys/build.rs

```diff
@@ -164,6 +168,13 @@ fn try_pkg_config() {
     // Otherwise if we're going to windows we probably can't use pkg-config.
     if target.contains("windows-gnu") && host.contains("windows") {
         env::set_var("PKG_CONFIG_ALLOW_CROSS", "1");
+    } else if target.contains("windows-gnu") && host.contains("linux") {
```

Comment:
This seems a bit over-opinionated about how a linux -> windows cross build will work. It's probably more reasonable for people doing this to set these environment variables themselves.